noir-projects/aztec-nr/aztec/src/state_vars/storage.nr

```diff
@@ -8,9 +8,8 @@ trait Storage<T> where T: Serialize<N> + Deserialize<N> {
 
 // Struct representing an exportable storage variable in the contract
 // Every entry in the storage struct will be exported in the compilation artifact as a 
-// Storable entity, containing the storage slot and the type of the variable
```

Comment:
The type wasn't being used anywhere (I just thought it would be nice to have it when I implemented it) and complicates call interfaces a lot. Removing for now